### PR TITLE
add docker compose config for dev workflow

### PR DESCRIPTION
**Pull request by mathroc**
I don’t know if you’re interested in having this in the repository, that’s the config I’m using to work on graphile in docker

### CODE REVIEW

File: config/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM node:8
+
+RUN apt-get update && apt-get install -y postgresql-client
+
+RUN npm install -g lerna yarm flow-bin
```
Comment (benjie) [inline config/docker/Dockerfile]: Should that be yarn?

**Comment by benjie** [comment]
Thanks for the PR; I'd prefer this as a wiki article for now (trying to keep everything as lean as possible), but we can revisit at a later date. Thanks!

File: config/docker/Dockerfile (hunk 1 above)
Comment (mathroc) [reply config/docker/Dockerfile]: ahh in fact it shouldn’t be there at all, you just made me realize I’ve installed another package by mistake ^^

**Comment by mathroc** [comment]
hmm I was going to take a look at [the wiki](https://github.com/graphile/graphile-build/wiki) but it redirect to the repository home 😕 or did you meant https://github.com/graphile/graphile.github.io ?

**Comment by benjie** [comment]
Try now, I've opened the permissions up I think 👍

**Comment by mathroc** [comment]
yes that worked : https://github.com/graphile/graphile-build/wiki/Development-with-docker-compose

thank you

**Comment by benjie** [comment]
Awesome! Really appreciate that, thanks @mathroc 👍 

If you want to patch the README to link to the wiki page that would be welcome too.

**Pull request closed without merging**